text/0008--individual-agency-company-pages.md

```diff
@@ -0,0 +1,131 @@
+* type -- Individual, a Digital Agency, or a Company. We use the word Organization below to refer to either a Digital Agency and Company.
+* a description of them in the third person “Eg, Mike Allanson is a developer living in London"
+* Personal website
+* GH handle (?)
+* Open for Work [boolean]
+* A photo / logo (depending on Individual / Organization type)
+* Org they are affiliated with (if Individual)
+* People they are affiliated with (if Org)
+* A list of open jobs (if they are Organization type)
+* A list of all the sites they’ve built.
+* A location with:
+    * A description (“London, UK”)
+    * Lat / lng
+
+_Data Format_: all creators should live in a creators.yml file, similar to sites.yml.
+
+**Jobs** have the following fields:
+* Title
+* Location
+* Organization (FK)
+* (Optional) salary low end
+* (Optional) salary high end
+* (Optional) salary currency
+* Contact email or application link. 
+* Excerpt (short version)
+* Description
+
+_Data Format_: Jobs should be stored in markdown files in a /jobs folder. Jobs should be stored in markdown files. Frontmatter should have all fields except description. Body of the file should be the description. 
+
+### Notes on long-term data strategy
+* Right now, the data is in YAML in the GitHub repo. At some point, we may move all of our data to a format with a UI making it easier for people to submit & edit their own data, without imposing a review burden on Gatsby maintainers
+* Option 1: use Airtable or a CMS
+    * Airtable doesn’t have an easy way to allow users to edit individual forms in a record. 
+    * We could build something on top with Lambdas, or utilize the Zapier => Airtable integration: https://support.airtable.com/hc/en-us/articles/115007655568-Using-forms-to-edit-tables-with-Zapier
+    * We could also make job postings go down after 60 days
+* Option 2: Build an application ourselves
+    * Could do this, but what’s the value here?
+    * We probably don’t want to spin up an application to do this
+    * Firebase as backend?
+* **Option 3: Stick with YAML in Github for now (what we’re doing)**
+    * We know this works, although it does add additional maintainer burden reviewing these files
+    * If we find a good global “application” type solution, we probably want to migrate the sites in the site showcase here as well. 
+    * **Almost certainly the right move for now.**
+* Option 4: Use seeker.company or another TBD service
+    * Upside: seeker.company provides a UI to allow people to create job postings, and a job board to list those postings. 
+    * Downside: No control over look and feel, not built with Gatsby, no way to customize submissions or link to organizations, no way to edit postings.
+# Drawbacks
+
+Why should we *not* do this? Please consider:
+
+- Time & effort
+- It will add more pages to .org to maintain
+- Maintainers will be getting PRs adding additional agencies / sites / companies
+- We could get spam submissions that degrade the site.
```

Comment:
hm, interesting and I think true. This project opens the community up to spammers OR even people who are actual programmers and maybe just aren't delivering for whatever reason, which could be disappointing to the people who hire them.

Reply:
I was thinking more spam in terms of people linking to spam sites, although I suppose that's actually more likely with the site showcase. Will remove this line.